tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -54,29 +58,105 @@ struct SynthOrdered {
     }
 };
 
-template <class T>
-inline constexpr bool is_pair = false;
-template <class A, class B>
-inline constexpr bool is_pair<std::pair<A, B>> = true; // TRANSITION, std::pair spaceship not yet implemented
+struct OrderedChar {
+    OrderedChar() = default;
+    OrderedChar(const char c) : c(c){};

```

Comment:
```suggestion
    OrderedChar(const char c) : c(c) {}
```

Reply:
Clang-format made me do it!

Reply:
🤨

Reply:
struct OrderedChar {
     OrderedChar() = default;
-    OrderedChar(const char c) : c(c) {};
+    OrderedChar(const char c) : c(c){};
 
     OrderedChar& operator=(const char& other) {
         c = other;

Reply:
```suggestion
    OrderedChar(const char c) : c(c){}
```